packages/core/__tests__/sagaHelpers/throttle.js

```diff
@@ -58,17 +59,11 @@ test('throttle', () => {
       // throttle must ignore incoming actions during throttling interval
       expect(actual).toEqual(expected)
     })
-    .then(
-      () => {
-        clock.uninstall()
-      },
-      err => {
-        clock.uninstall()
-        throw err
-      },
-    )
 })
+
 test('throttle: pattern END', () => {
+  jest.useRealTimers()
```

Comment:
cleanup should be done in the same test when you setup a  fake timer (as it's done before)